lms/static/js/courseware/bridge.js

```diff
@@ -0,0 +1,58 @@
+function sendMessageToIOS(message) {
+  window?.webkit?.messageHandlers?.IOSBridge?.postMessage(message);
+}
+
+function sendMessageToAndroid(message) {
+  window?.AndroidBridge?.postMessage(message);
+}
+
+function markProblemCompletedIOS(message) {
```

Comment:
There's no need to do these 2 proxy calls if there are no differences in IOS and Android calls

Reply:
Yes, there are none at the moment (although we haven't had any feedback from IOS yet), but if in the future there is a need for this, it will allow us to make changes to only one function, without having to change the names on the mobile side.

Reply:
I believe the name of the function doesn't mean much we should relly on JS Event names, am I correct?

> window.addEventListener("messageFromIOS"

Reply:
I'l recheck this.